libs/spark/src/theme/theme.ts

```diff
@@ -92,6 +92,7 @@ declare module '@material-ui/core/styles/createPalette' {
 export const prendaTheme: ThemeOptions = {
   typography: {
     fontFamily: '"Nunito", Avenir, sans-serif',
+    fontSize: 1,
```

Comment:
What is `1` referencing? Should this be `16` 
Or maybe better:
```
    htmlFontSize: 16,
``` 
or
```
    bodyFontSize: 16,
```

Reply:
Turns out it's pixels. I thought it was `rem` since it only accepts a number not a string (TS complains), but turns out the engine converts the px value to rem behind the scenes. We won't want to override `htmlFontSize` because of the a11y detriment noted on your link, but 16 is the correct value for `fontSize`